Lib/configparser.py

```diff
@@ -1040,6 +1042,33 @@ def __iter__(self):
         # XXX does it break when underlying container state changed?
         return itertools.chain((self.default_section,), self._sections.keys())
 
+    def __str__(self):
+        config_dict = {
+            section: dict(self.items(section)) for section in self.sections()
```

Comment:
This performs interpolations on all of the values, which can raise `InterpolationError` (or other arbitrary exceptions, depending on what `interpolation` class was specified). I think generally `__str__` should be "safe" and not perform operations that can fail unexpectedly